tests/yam/agama/boot_encrypted_root.pm

```diff
@@ -0,0 +1,30 @@
+## Copyright 2024 SUSE LLC
```

Comment:
`ENCRYPT=1` avoids to create new file, I think we can think about how to do this in the future.